lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -569,50 +573,54 @@ static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
 
   if (!self_type.IsValid() ||
       !self_type.GetTypeSystem()->SupportsLanguage(lldb::eLanguageTypeSwift))
-    return;
+    return Status("Could not add required aliases: self is not valid.");
 
   // Import before getting the unbound version, because the unbound
   // version may not be in the mangled name map.
 
   CompilerType imported_self_type = ImportType(swift_ast_context, self_type);
 
   if (!imported_self_type.IsValid())
-    return;
+    return Status(
+        "Could not add required aliases: imported self type is not valid.");
 
   auto *stack_frame = stack_frame_sp.get();
   if (bind_generic_types != lldb::eDontBind) {
     imported_self_type = swift_runtime->BindGenericTypeParameters(
         *stack_frame, imported_self_type);
     if (!imported_self_type)
-      return;
+      return Status("Could not add required aliases: could not bind self "
+                    "type's type parameters.");
   }
 
   {
     auto swift_type_system =
         imported_self_type.GetTypeSystem().dyn_cast_or_null<TypeSystemSwift>();
     if (!swift_type_system)
-      return;
+      return Status("Could not add required aliases: unexpected type system.");
 
     // This might be a referenced type, in which case we really want to
     // extend the referent:
     imported_self_type = swift_type_system->GetReferentType(
         imported_self_type.GetOpaqueQualType());
     if (!imported_self_type)
-      return;
+      return Status("Could not add required aliases: could not get self's "
+                    "referent type.");
   }
 
   {
     auto swift_type_system =
         imported_self_type.GetTypeSystem().dyn_cast_or_null<TypeSystemSwift>();
     if (!swift_type_system)
-      return;
+      return Status("Could not add required aliases: unexpected type system.");
 
     // If we are extending a generic class it's going to be a metatype,
     // and we have to grab the instance type:
     imported_self_type = swift_type_system->GetInstanceType(
         imported_self_type.GetOpaqueQualType());
     if (!imported_self_type)
-      return;
+      return Status(
+          "Could not add required aliases: could not get self instance type.");
```

Comment:
```suggestion
          "Unable to add the aliases the expression needs because the Swift expression parser couldn't get the instance type for `self`.");
```

If the word "Unable" isn't a valid option, then I suggest the contraction "Couldn't".